test/package.e2e-spec.ts

```diff
@@ -73,51 +72,50 @@
     afterAll(async () => {
         await app.close();
     });
+});
+
+describe('Retry logic tests', () => {
+    it('retry throws exception on timeout', async () => {
+        // @ts-ignore
+        const retryFunction = undefined;
+
+        let exceptionFound = false;
+
+        try {
+            await ProtocolUtility.retryForDuration(2000, 500, retryFunction);
+        } catch (e) {
+            exceptionFound = true;
+        }
+
+        expect(exceptionFound).toBe(true);
+        return;
     });
 
-    describe('Retry logic tests', () => {
-        it('retry throws exception on timeout', async () => {
```

Comment:
This test needs to remain.  I think the confusing is we have 2 tests named the same but they are not actually the same.

Reply:
Ahh I didn't actually delete anything in this file, it's just spacing changes and removing a couple unused local variables in the `beforeAll` block. The way github shows changes is weird.

Reply:
my apologizes.  I thought it was strange. lgtm